src/main/json/r6OperatorCounters.json

```diff
@@ -1503,6 +1503,33 @@
                     "properties": {
                         "Description": "Dokkaebi can use her Logic Bomb to ring Caveria's phone. This will reveal Caveria's position when roaming even if Silent Step is active."
                     }
+                },
+                {
+                  "id": "143",
+                  "type": "Hard Counter",
+                  "startNode": "48".
+                  "endNode": "47",
```

Comment:
The git diff is flagging up something about the colon in this line. It looks fine to me but would you be able to check it works okay, please?